src/analysis/static/graph/parsing/TargetMetaData.ts

```diff
@@ -0,0 +1,7 @@
+
+export interface TargetMetaData {
```

Comment:
Maybe, we should also make a class/interface for the path and the name of a target since these two should always be together to identify a target. We could call this class TargetIdentifier or TargetName

Reply:
We already have that in the TargetFile & Target classes